ui/imports/shared/popups/send/controls/TokenBalancePerChainDelegate.qml

```diff
@@ -98,7 +98,7 @@ StatusListItem {
             asset.width: 16
             asset.height: 16
             asset.isImage: true
-            asset.name: Style.svg("tiny/%1".arg(iconUrl))
+            asset.name: Style.svg("tiny/%1".arg(model.iconUrl || "network/Network=Testnet"))
```

Comment:
why are we hardcoding his?

Reply:
We don't really have to but I was getting a lot of warnings there

Reply:
Removed